annotated-projects/PXD006195/sdrf.tsv

```diff
@@ -0,0 +1,73 @@
+source name	characteristics[organism]	characteristics[organism part]	characteristics[cell type]	characteristics[disease]	assay name	comment[fraction identifier]	comment[label]	comment[label]	comment[data file]	comment[associated file uri]	comment[instrument]	comment[dissociation method]	comment[cleavage agent details]	comment[fragment mass tolerance]	comment[fragment mass tolerance]	comment[modification parameters]	comment[modification parameters]	comment[modification parameters]	factor value[label]	factor value[temperature]
```

Comment:
I guess the labels should be
AC=PRIDE:0000611;NT=SILAC light R:12C(6)14N(4)	AC=PRIDE:0000613;NT=SILAC light K:12C(6)14N(2)
here (and then every other line)

Reply:
I suppose your right. I created a fix.